### ISSUE TITLE
Implement ArrayAccess

### ISSUE THREAD

**Issue by chrisvanpatten** [shameerc/Slim3-Aura.Di-Bridge#1]
Not sure if this is the right place, but it might be useful to implement ArrayAccess, so you can access injected dependencies via `$container['dependency']`, as with the default Slim container.

I've noticed a few Slim extensions that (perhaps wrongly) make the assumption that the default Slim container is being used, and try to access dependencies via array.

Using Aura's get/set methods would/should still be the preferred method, but this would provide some extra compatibility.

**Comment by shameerc** [comment]
I agree that implementing array access will be a useful feature. I will have a look if it's possible to implement :)
